jsonnet/rules.libsonnet

```
@@ -343,15 +350,15 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
           },
           {
             expr: |||
-              (
+              (((
                 kube_deployment_spec_replicas{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}
-                  !=
+                  >
                 kube_deployment_status_replicas_available{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}
               ) and (
                 changes(kube_deployment_status_replicas_updated{namespace=~"(openshift-.*|kube-.*|default|logging)",job="kube-state-metrics"}[5m])
                   ==
                 0
-              ) and cluster:control_plane:all_nodes_ready
+              )) * scalar(cluster:control_plane:all_nodes_ready)) != 0
```

(nit) I found it easier to parse in this form:

```suggestion
              )) * scalar(cluster:control_plane:all_nodes_ready == 1)
```